packages/mui-joy/src/Avatar/Avatar.tsx

```diff
@@ -166,7 +166,8 @@ const Avatar = React.forwardRef(function Avatar(inProps, ref) {
   } = props;
   const variant = inProps.variant || groupContext?.variant || variantProp;
   const { getColor } = useColorInversion(variant);
-  const color = getColor(inProps.color || groupContext?.color, colorProp);
+  const colorFromContext = inProps.color || groupContext?.color;
+  const color = colorFromContext !== 'context' ? getColor(colorFromContext, colorProp) : colorProp;
```

Comment:
What's the issue that cause this change?

Reply:
`colorFromContext` can be `'context'` while `getColor` doesn't accept `'context'`.
![Screenshot 2023-04-05 at 12 17 03](https://user-images.githubusercontent.com/32841130/230065166-49e83191-02aa-4164-80d1-29df640c091b.png)